Lib.Web.Mvc/packages.config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.AspNet.Mvc" version="5.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.Razor" version="3.2.3" targetFramework="net461" />
-  <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net461" />
+  <package id="Microsoft.AspNet.Mvc" version="5.2.7" targetFramework="net461" />
```

Comment:
I give this a little bit of thinking and it might be better to migrate to PackageReference. This would allow a floating version (5.2.*).

Reply:
Do you plan to implement that then?

Maybe merge (and release?) this for now, until that happens?

Reply:
The trick is, that this change is a breaking one while PackageReference isn't.

I'll try to take care of that tonight.

Reply:
So...

Pushing a signed .NET Framework package after ~3 years can be more challenging than one can suspect, but it should be out there (I hope).

Reply:
Looks good, I don’t see the mismatch warning anymore. Thank you :)